ml-agents/mlagents/trainers/cli_utils.py

```diff
@@ -169,10 +169,10 @@ def _create_parser() -> argparse.ArgumentParser:
         help="Forces training using CPU only",
     )
     argparser.add_argument(
```

Comment:
I think we should still have --torch but raise a warning saying it is useless and it will be removed in a future release.
Similarly, we should raise a warning if the user is using --tensorflow as it will be deprecated in a future release.

Reply:
Added the warnings and --torch back